protolator/json.go

```diff
@@ -315,20 +311,28 @@ func protoFields(msg proto.Message, uMsg proto.Message) ([]protoField, error) {
 	}
 
 	iResult := make([][]protoField, len(fieldFactories))
-
-	protoProps := proto.GetProperties(mVal.Type())
+	t := mVal.Type()
 	// TODO, this will skip oneof fields, this should be handled
 	// correctly at some point
-	for _, prop := range protoProps.Prop {
-		fieldName := prop.OrigName
-		fieldValue := mVal.FieldByName(prop.Name)
-		fieldTypeStruct, ok := mVal.Type().FieldByName(prop.Name)
+	for i := 0; i < t.NumField(); i++ {
```

Comment:
I'm sorry, what do you see as a different approach?

Functions were used here that are no longer present in the new protobuf. I was just trying to replicate the logic by other means. Although in my opinion, this code fragment is very overcomplicated. 

I think it can be written in a simpler way. But not in this pr.

Reply:
proto.GetProperties is missing from the new protobuf
I got inside her and tried to replicate her logic here

Reply:
When I rewrote it, I got it wrong 3 or 4 times. And the tests didn't pass. This is the result of long debugging.

But if there is something wrong here, tell me what. I'll rewrite it.

Reply:
I meant why did you have to utilize a new approach versus the prior code. I understand now. I've also tested the fabric consumers such as `configtxgen -inspect` with the new PR, and it works fine, so I'll go ahead and merge.